11LessonEleven/homework/css/inspired.css

```diff
@@ -0,0 +1,98 @@
+main{
+  font-size: serif;
+}
+
+.headline{
+  background-image: url("../img/bg2.jpg");
+  background-attachment: fixed;
+  background-size: cover;
+  color:#fff;
+  display: flex;
+  flex-flow:column nowrap;
+  align-items: flex-start;
+  justify-content: center;
+  height: 500px;
+}
+
+.headline h1{
+  margin-left: 100px;
+  width:400px;
+  font-size: 3.5em;
+}
+
+ .headline p{
+   margin-left:100px;
+   width:150px;
+ }
+
+.main-content{
+  display:flex;
+  flex-flow:row wrap;
+  align-items: baseline;
+  justify-content: center;
+}
+
+.main-content h2{
+  margin:90px 20px 50px 20px;
+  font-size: 1.25em;
+  font-weight: 500;
+}
+
+.main-content p{
+  margin:0 20px 10px 20px;
+  font-size: .75em;
+  color:gray;
+  line-height: 1em;
+}
+article{
+  width:90%;
+}
+
+/*-----Media Queries----*/
+@media screen and (min-width: 320px){
+  main::before {
+    content: "320px";
+    color: green;
+    position: absolute;
+    top: 0;
+    left: 0;
+    font-weight: 700;
+    margin: 5px;
+    font-size: 2em;
+  }
+
+}
+@media screen and (min-width: 480px){
+  main::before {
+    content: "480px";
+    color: maroon;
+  }
+}
+@media screen and (min-width: 620px){
+  main::before {
+   content: "620px";
+   color: orange;
+ }
+  .headline{
+    background-image: url("../img/bg.jpg");
+  }
```

Comment:
Try anchoring the image to show us the interesting parts. Experiment with `background-position:right top;`. http://www.w3schools.com/css/tryit.asp?filename=trycss_background-image_position